### ISSUE TITLE
Error during source extraction when using conan install --build with a specific file

### ISSUE THREAD

**Issue by claasd** [conan-io/conan#1693]
Hi,

I have the following error:
```
PROJECT: Installed build requirements of: myOrg.SomeComponent/2.14.0-alpha-build.5@user/unstable
Downloading conan_sources.tgz

Traceback (most recent call last):
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\command.py", line 884, in run
    method(args[0][1:])
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\command.py", line 268, in install
    filename=args.file, cwd=args.cwd)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\conan_api.py", line 57, in wrapper
    return f(*args, **kwargs)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\conan_api.py", line 399, in install
    no_imports=no_imports)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\manager.py", line 347, in install
    installer.install(deps_graph, current_path)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\installer.py", line 135, in install
    self._build(nodes_by_level, skip_private_nodes, deps_graph)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\installer.py", line 222, in _build
    self._remote_proxy.get_recipe_sources(conan_ref, conan_file.short_paths)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\proxy.py", line 123, in get_recipe_sources
    current_remote)
  File "d:\buildFolder\virtualenv\lib\site-packages\conans\client\remote_manager.py", line 199, in get_recipe_sources
    shutil.rmtree(c_src_path)
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 478, in rmtree
    return _rmtree_unsafe(path, onerror)
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 368, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 368, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 368, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 368, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 373, in _rmtree_unsafe
    onerror(os.unlink, fullname, sys.exc_info())
  File "d:\buildFolder\virtualenv\lib\shutil.py", line 371, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Access is denied: 'd:\\buildFolder\\data/.cn\\vy4r6yxd\\1\\.c_src\\SomeComponent\\doc\\QA\\Old\\Testdrehbuch-SomeProject.xls'

ERROR: [WinError 5] Access is denied: 'd:\\buildFolder\\data/.cn\\vy4r6yxd\\1\\.c_src\\SomeComponent\\doc\\QA\\Old\\Testdrehbuch-SomeProject.xls'
```

This error happens on every build, on Jenkins machines as well as on my developer machine. The file is not large (~80Kb), nor does it have special characters as far as I can see,
Any idea how this might happen?

conan version: 0.26.0 (also appears on 0.25.0)
Windows (Server 2012 / Win7)
Python 3.5.2
[log-developer-machine.txt](https://github.com/conan-io/conan/files/1269481/log-developer-machine.txt)

**Comment by lasote** [comment]
I've no idea. Do you have the windows indexer enabled?
You could try to access that file manually and see what happen, or check if any process is blocking it.

**Comment by claasd** [comment]
No windows indexer, special folder, no idea how this can happen. 
I will try a similar solution to #1295

**Comment by memsharded** [comment]
You might try also replacing:

```diff
diff --git a/conans/client/remote_manager.py b/conans/client/remote_manager.py
index a5a6ca5..b115502 100644
--- a/conans/client/remote_manager.py
+++ b/conans/client/remote_manager.py
@@ -196,7 +196,7 @@ class RemoteManager(object):
         c_src_path = os.path.join(export_sources_folder, ".c_src")
         if os.path.exists(c_src_path):
             merge_directories(c_src_path, export_sources_folder)
-            shutil.rmtree(c_src_path)
+            rmdir(c_src_path)
         for dirname, _, filenames in os.walk(export_sources_folder):
             for fname in filenames:
                 touch(os.path.join(dirname, fname))
```

**Comment by claasd** [comment]
@memsharded : The fix worked! Would it be possible to include it in an upcoming 0.26.1?

**Comment by claasd** [comment]
I just realized it is not that urgent. I can exclude those files in the export_sources command.

**Comment by lasote** [comment]
Sure, if we open 0.26.1 it could be included. No much bugs so far.

**Comment by memsharded** [comment]
A minor branch has been created, included this.
https://github.com/conan-io/conan/tree/release/0.26.1

**Comment by memsharded** [comment]
This has been released in 0.26.1, please update and try.
